src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -29,13 +29,29 @@ public class Bundler
         readonly TargetInfo Target;
         readonly BundleOptions Options;
 
-        // Assemblies are 16 bytes aligned, so that their sections can be memory-mapped cache aligned.
-        public const int AssemblyAlignment = 16;
+        // Temporary overload to avoid breaking the SDK build.
```

Comment:
Don't we want to break the SDK build? That is, don't we want the insertion that brings this into the SDK to be accompanied with our change on the SDK side?

Reply:
It would block dependency flow from runtime -> SDK until that change went in, which might be nice to avoid in case there are other fixes going through. If temporarily blocking it is ok, I'll remove this.

Reply:
It will only block ingestion of 6.0 bits right? That seems fine to me.

Reply:
Removed this as discussed. The plan is to make the SDK change quickly so that the dependency flow is not blocked for long.

Reply:
We also need the change migrated to 5.0 branches in both repos, isn't it? Can we briefly take breaking changes there too? 
In SDK, it is typical to only checkin to the 5.0 branch release, and let the change flow into master branch.

Reply:
Not sure I understand what's breaking about this change, aside from the fact that the SDK needs a parallel fix -- but it's not like the SDK can not take that fix, fixing the bug means changing both repos.

Reply:
The SDK's invocation of the bundler needs to pass in the architecture argument, but it can only do so when this change flows into the SDK through HostModel version update. Given the position of `Architecture` argument, this looks like it'll break the SDK integration job; Isn't it?. So, we have the following options: 
* Make the update as part of SDK integration commit
* Make the update in two steps -- once SDK gets a chance to update.

In this case, the change is small, and its OK to follow the former approach. But, we've followed the latter approach so far for HostModel/SDK interdependencies (originally per Nick's recommendation).

Reply:
I'm not sure why he suggested (2), but I rather prefer (1), as it removes the possibility of accidentally dropping the corresponding change somewhere.

Reply:
Nick suggested (2) because the SDK parts of the change are typically non-trivial, and they could be independently reviewed in a normal PR, rather than in the automated integration job, and because they can be independently reverted if there's a problem without affecting the other parts of integration.

But in this case, the change on the SDK side is very small, so I'm fine with (1).

Reply:
That makes sense. For larger changes (2) could definitely be appropriate, especially if we're concerned that CI wouldn't fully validate the change.

Reply:
Just to be clear - I do plan to get the change reviewed in a normal PR, not in the automated job. I'll include a manual dependency update in my PR, but the automated flow will be blocked until my PR is merged.